### PR TITLE
#190: Implement complete LDAP based login

### PR DESCRIPTION
**Pull request by bees4ever**

### CODE REVIEW

File: ws/src/main/scala/de/thm/ii/submissioncheck/misc/LDAPConnector.scala [hunk 1]
```diff
@@ -1,25 +1,69 @@
...
 
 /**
   * Using ldaptive to simple register to THM LDAP Service
+  *
   * @author http://www.ldaptive.org/
   */
-object LDAPConnector
-{
+object LDAPConnector {
+  private val LDAPURL = "ldaps://ldap.fh-giessen.de"
```
Comment (andrej-sajenko) [inline ws/src/main/scala/de/thm/ii/submissioncheck/misc/LDAPConnector.scala]: Can you move the real URL into the resources/application.properties file and read it from there, please.

File: ws/src/main/scala/de/thm/ii/submissioncheck/controller/LoginController.scala [hunk 2]
```diff
@@ -94,30 +95,34 @@ class LoginController extends CasClientConfigurerAdapter {
...
     * @author Benjamin Manns
-    * @param request Http request gives access to the http request information.
+    * @param request  Http request gives access to the http request information.
     * @param response HTTP Answer (contains also cookies)
     * @param jsonNode Request Body of User login
     * @return Java Map
     */
   @RequestMapping(value = Array("login/ldap"), method = Array(RequestMethod.POST))
   def userLDAPLogin(request: HttpServletRequest, response: HttpServletResponse, @RequestBody jsonNode: JsonNode): Map[String, Boolean] = {
     try {
```
Comment (andrej-sajenko) [inline ws/src/main/scala/de/thm/ii/submissioncheck/controller/LoginController.scala]: The indentations look strange. Can you unify them.

File: ws/src/main/scala/de/thm/ii/submissioncheck/misc/LDAPConnector.scala [hunk 3]
```diff
@@ -1,25 +1,69 @@
...
+    *
     * @author Benjamin Manns
     * @param uid user Id where we want to get more informations /attributes of
     * @return Attribute Set
     */
   def loadLDAPInfosByUID(uid: String): LdapEntry = {
-    val cf = new DefaultConnectionFactory("ldap://ldap.fh-giessen.de")
+    val cf = new DefaultConnectionFactory(LDAPURL)
     val executor = new SearchExecutor()
     executor.setBaseDn("dc=fh-giessen-friedberg,dc=de")
```
Comment (andrej-sajenko) [inline ws/src/main/scala/de/thm/ii/submissioncheck/misc/LDAPConnector.scala]: Base DN should also be moved into application.properties.

**Comment by bees4ever** [comment]
@andrej-sajenko  I changed it, hope it is ok now?

**Pull request merged**